src/Plugin/StrawberryRunnersPostProcessor/OcrPostProcessor.php

```diff
@@ -484,17 +484,17 @@ protected function hOCRtoMiniOCR($output, $pageid) {
               $y0 = (float) $wcoos[2];
               $x1 = (float) $wcoos[3];
               $y1 = (float) $wcoos[4];
-              $l = round(($x0 / $pwidth), 3);
-              $t = round(($y0 / $pheight), 3);
-              $w = round((($x1 - $x0) / $pwidth), 3);
-              $h = round((($y1 - $y0) / $pheight), 3);
+              $l = ltrim(sprintf('%.3f',($x0 / $pwidth)), 0);
```

Comment:
Beautiful friend!! Should we also check if there are 4 coordinates? Or do we expect we will always have them? Open to your opinion since OCR is your domain

Reply:
`sprintf` is the way!

Reply:
I think we always have 4 coords, that depends on tesseract/dejvu2hocr but to be really safer we can add that check and discard words without 4 coords. I don't think that waste a lot of time more, so for me ok

Reply:
About check coords, I have to notes:

Reply:
Up to you friend. This is all for highlights but also for text extraction so maybe if we have 3 we can deduce the missing part and if not then probably the OCR is already wrong (e.g detecting a word like ------) so safe to skip?

Reply:
My doubt is: if we have only 3 coords that means something went wrong so better no calculation and skip word?
So, to be safer, when not 4 coords then we have to discard all word included. Do you agree?

Reply:
@DiegoPino I added the check about number of coords, please also double check, for me ok

Reply:
Perfect!!